lib/mayaUsd/ufe/ProxyShapeHierarchy.h

```diff
@@ -98,7 +98,8 @@ class MAYAUSD_CORE_PUBLIC ProxyShapeHierarchy : public Ufe::Hierarchy
 
 private:
     const PXR_NS::UsdPrim& getUsdRootPrim() const;
-    Ufe::SceneItemList     createUFEChildList(const PXR_NS::UsdPrimSiblingRange& range) const;
+    Ufe::SceneItemList
+    createUFEChildList(const PXR_NS::UsdPrimSiblingRange& range, bool filterInactive) const;
```

Comment:
Negative boolean tend to be confusing. I would prefere this to be `filter` with true meaning do filter, false meaning do not filter.

Reply:
Oh, it mean filter-the-inactive-nodes and not make-filter-inactive... I mis-read the meaning of the variable.